.gitignore

```diff
@@ -17,6 +17,9 @@
 # emacs
 flycheck*
 
+## Visual code settings
+.vscode/
+
```

Comment:
```suggestion
```
Can you remove this one then, since it is now at the end of the file?

Reply:
Done!